app/models/endpoint.rb

```diff
@@ -77,4 +77,8 @@ def validate_certificate_authority
   rescue OpenSSL::OpenSSLError => err
     errors.add(:certificate_authority, err.to_s)
   end
+
+  def should_validate_url?
+    !resource.kind_of?(ManageIQ::Providers::CloudManager)
```

Comment:
How about moving this to a method on the `ExtManagementSystem` so we can just override it in `manageiq/providers/cloud_manager` ?

Reply:
It would give us a more generic way of doing the same thing for other provider types without getting into a long list of `kind_of? || kind_of? ...`

Reply:
Problem that `resource` is polymorphic. It means, we have no guarantee that it will have such method. However, that can be verified, so, ok.